### ISSUE TITLE
Can I put my face photo in your pre-trained  web_ Demo to generate a  3D？ video

### ISSUE THREAD

**Issue by lemonsstyle** [PeterouZh/CIPS-3D#37]
Hello, thank you for your contribution. I try to run your web_ Demo, it seems to be about the pre-trained of face data sets. Can I import other face images to produce the effect like the demo-video? I can't find where to put my face photo, thank you.

**Comment by PeterouZh** [comment]
Yes, you can. Please use this project [https://github.com/PeterouZh/CIPS-3Dplusplus](https://github.com/PeterouZh/CIPS-3Dplusplus).
